velox/exec/Operator.cpp

```diff
@@ -52,11 +52,18 @@ OperatorCtx::createConnectorQueryCtx(
     memory::MemoryPool* connectorPool,
     memory::SetMemoryReclaimer setMemoryReclaimer,
     const common::SpillConfig* spillConfig) const {
+  auto connectorConfig =
+      driverCtx_->task->queryCtx()->getConnectorConfig(connectorId);
+  VELOX_CHECK(connectorConfig);
+  if (connectorConfig->empty()) {
```

Comment:
Why do we only pass in connectorProperties only when connectorConfig (from session property) is empty?

Also not sure if we should just merge connectorConfig and connectorProperties in one data structure, it would look confusing to have
`"hive.orc.writer.stripe-max-size`
`orc_optimized_writer_max_stripe_size`
int the same data structure.

Reply:
@kewang1024 you are right. We should ideally normalize and merge them. I will fix this.
I think the goal should be to normalize the connectorConfig at a higher level say here. We can use the session property name as the normalized name as they take precedence. In your example, it would be `orc_optimized_writer_max_stripe_size`.
This is useful to avoid passing both session, config parameters as in this PR
https://github.com/facebookincubator/velox/pull/6815/files#r1341007336